controller/gui/client.js

```diff
@@ -65,3 +61,57 @@ customElements.define(
   },
   { extends: 'input' }
 )
+
+/* Form web component preventing more than one submission.
+ *
+ * Disable submit input inside the form after the first submission.
+ */
+customElements.define(
+  'disable-after-submit',
+  class extends HTMLFormElement {
+    constructor() {
+      super()
+
+      const form = this
+      const button = form.querySelector('input[type=submit]')
+
+      const buttonParent = installParent(button, document.createElement('span'))
+      buttonParent.style = `
+        position: relative;
+        height: fit-content;
+      `
+
+      const spinnerParent = document.createElement('div')
+      spinnerParent.style = `
+        display: flex;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+      `
+
+      const spinner = document.createElement('span')
+      spinner.className = 'd-Spinner'
+      spinnerParent.appendChild(spinner)
+
+      form.addEventListener('submit', function() {
+        button.disabled = true
+        button.style.color = 'transparent'
+        button.className += ' d-Button--Disabled'
```

Comment:
This `className` is not being referenced in `style.css`.

Reply:
https://github.com/dividat/playos/pull/83/commits/43128e7a5b331874e66b266181da9c2948b6e3ff